### PR TITLE
Add all of them missing meeting minutes 😎

### PR DESCRIPTION
**Pull request by Starefossen**
**TODO**

* [ ] Go through issue tracker and find missing meetings

**New meeting minutes:**

* doc/meetings/build-wg-meeting-2015-05-14.md
* doc/meetings/build-wg-meeting-2015-05-25.md
* doc/meetings/build-wg-meeting-2015-06-09.md
* doc/meetings/build-wg-meeting-2015-06-23.md
* doc/meetings/build-wg-meeting-2015-07-28.md
* doc/meetings/build-wg-meeting-2015-08-18.md
* doc/meetings/build-wg-meeting-2015-08-25.md
* doc/meetings/build-wg-meeting-2015-09-29.md
* doc/meetings/build-wg-meeting-2015-11-10.md

### CODE REVIEW

**Comment by rvagg** [comment]
lgtm, thanks @Starefossen

File: doc/meetings/build-wg-meeting-2015-04-15.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+* @rvagg: Continue to work with automation of Pull Request and feedback
+
+### Configure Scripts
+
+* Build WG to take on ownership of the io.js configure scripts in core
+
+**Action Items:**
+
+* @ravagg: To document this in the Build WG Governacne document
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-04-15.md]: typo

File: doc/meetings/build-wg-meeting-2015-04-15.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* Johan Bergström (@jbergstroem)
+* Rod Vagg (@rvagg)
+
+### Build Respources
+
+This is documented here: https://github.com/nodejs/build/issues/77
+
+* Most are Linux machines running on DigitalOcean + Jenkins + iojs.org
+* RackSpace does all the Windows stuff (7 servers)
+* Voxer does all the FreeBSD (2 server) and Mac stuff (2 Mac Mini)
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-04-15.md]: server -> servers

File: doc/meetings/build-wg-meeting-2015-05-25.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  - Azure
+* Jenkins + Windows uncertainty
+  - Flakey tests?
+* Secrets and asset management
+* Group interaction going forward
+
+## Present
+
+* Alexis Campala
+* Forrest Norvel
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-05-25.md]: *Norvell

File: doc/meetings/build-wg-meeting-2015-05-25.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+* Rod Vagg
+* Johan Bergstrom
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-05-25.md]: *Bergström

File: doc/meetings/build-wg-meeting-2015-05-14.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+* How do you guys collaborate? Do you have meetings? On Australian time? :)
+
+### Info on node-accept-pull-request:
+
+* A bit of documentation is here:
+https://nodejs.org/documentation/workflow/#index_md_using_jenkins_to_build_test_and_merge_every_pr
+* Pretty much 0 test regressions since we adopted it
+* Flaky tests removed in io. PR: https://github.com/iojs/io.js/pull/812
+* Need to do a better job at documenting how this works. But let’s first agree
+on the high level plan * High priority improvements:
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-05-14.md]: Lacks a newline I think

**Comment by jbergstroem** [comment]
Few minor nits. "Present" seems to be somewhat inconsistent -- my suggestion would be to use Full Name (@username) everywhere.

**Comment by Starefossen** [comment]
@jbergstroem I have added all the docs I have access to from my Google Docs account. I have also re-formatted any existing documents to mach the the current format.

**Comment by rmg** [comment]
It shows my horrible attendance record, but otherwise LGTM :blush:

File: doc/meetings/build-wg-meeting-2015-06-23.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+* Johan
+  * lots of ccache work, init scripts all round, experimental clang on Windows
+    work
+* Alexis:
+  * CI for joyent/node (needs an lgtm
+    https://github.com/joyent/node/pull/25541), working on accept-pr for Jenkins
+* Michael
+  * PPC Jenkins nearly done
+* Rod
+  *
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2015-06-23.md]: empty, perhaps remove? missing?

**Comment by jbergstroem** [comment]
Thanks for doing this. Minor nit but LGTM. I haven't cross-referenced this with the documents at google docs though.

**Comment by orangemocha** [comment]
Rubber stamp LGTM. Thanks for doing this!

**Comment by mhdawson** [comment]
Anything holding up landing this ? If not I can go ahead and land it

**Comment by Starefossen** [comment]
I don't think there is anything in particular holding up this PR other than consistent formatting between meetings.

**Comment by jbergstroem** [comment]
..there's likely a few more to be fixed as well.

**Comment by jbergstroem** [comment]
..if anyone is up for it, obviously. Still +1 to land this.

**Comment by mhdawson** [comment]
If looks like @Starefossen had fixed up the consistency of the attendees.  I'd vote we land and then try to be more consistent going forward

**Comment by mhdawson** [comment]
Landed as 6ae85b8ec977dfd198a4f376d7353b7db970046a

**Comment by Starefossen** [comment]
@mhdawson PR is up for the remaining meeting minutes at #388

**Pull request closed without merging**